packages/site/src/ui/prop-types.js

```diff
@@ -20,12 +24,10 @@ const PropTypesRow = props => (
     <Table.Cell>
       <code className="nameCode">{props.row.name}</code>
       {props.row.required ? (
-        <Badge
-          appearance={Badge.appearances.stroke}
-          css={{ marginLeft: core.layout.spacingXXSmall }}
-        >
-          Required
-        </Badge>
+        <React.Fragment>
+          <BadgeSpacer />
```

Comment:
Why the spacer approach instead of just adding the margin?  (The added margin has a cleaner dom.) I'm fine with the spacer, btw.

Reply:
I guess i just like the way it reads and I prefer to not override the styles on the badge if possible